spec/lib/dotiw_spec.rb

```diff
@@ -101,11 +101,6 @@
         expect(DOTIW.languages.map(&:to_s).sort).to eq languages.sort
       end
 
-      it 'raises ArgumentError when nil is passed for time' do
-        expect { distance_of_time_in_words(nil) }.to raise_error(ArgumentError)
-        expect { distance_of_time_in_words(nil, nil) }.to raise_error(ArgumentError)
-      end
```

Comment:
This is an API change, same as above I would revert.

Reply:
Sorry I mean to move these down into the `with mixed inputs` block - these will still throw the `ArgumentError` as before.